src/Illuminate/Validation/Rules/File.php

```diff
@@ -86,7 +86,7 @@ class File implements Rule, DataAwareRule, ValidatorAwareRule
      * If no arguments are passed, the default file rule configuration will be returned.
      *
      * @param  static|callable|null  $callback
-     * @return static|null
+     * @return static|void
```

Comment:
This might be better suited for a separate PR as this is not strongly related to the rest of this PR